src/app/zone-egresses/views/item/IndexView.vue

```diff
@@ -1,13 +1,19 @@
 <template>
   <RouteView
-    v-slot="{ route }"
+    v-slot="{ route, can }"
     name="zone-egress-detail-tabs-view"
     :params="{
       zoneEgress: ''
     }"
   >
     <AppView
       :breadcrumbs="[
+        ...(can('use zones') ? [{
+          to: {
+            name: 'zone-cp-list-view',
+          },
+          text: t('zone-cps.routes.item.breadcrumbs')
+        }] : []),
```

Comment:
Only show the `Zone Control Planes >` breadcrumb in the Egress listing if zones are enabled.